opentelemetry-semantic-conventions/src/semconv/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod resource;
```

Comment:
Do we need semconv directory along with the generated files here, now that we are directly modifying the existing generated files ?

Reply:
No we don't need the dir. Removed it my recent force push. Will take off the WIP soon once I double check the tests and other code.